src/helpers/misc/Animal.re

```diff
@@ -379,6 +391,11 @@ let getStoryParagraphs = animal =>
       "It was a tough and trying time in Turin, Italy. A pandemic had taken over the land and the villagers were facing hardships. Unexpectedly nature was prospering, the villagers were in hiding and the flora and fauna of the region was reclaiming its land. Most unexpectedly the dragon returned, which for a long time was believed to be extinct. Glen the Dragon of Turin emerged from the Alps just North of the city. One of the last of his kind.",
       "Glen is a unique Wildcard that is named after Glen Weyl the author of Radical Markets. He was developed during the ETHTurin hackathon. The funds raised by Glen go to the conservation voted by the community. Owners of wildcards can vote using their loyalty tokens using quadratic voting to vote which conservation should receive the funds raised for that month.",
     |]
+  | Hook => [|
+      "Hook is a male blue whale that the Great Whale Conservancy has been keeping track of for a long time,close to 40 years. They have seen and photographically identified Hook for the past 6 years in the row which is a record siting for the Great Whale Conservancy. Hook has quite a dark pigmentation for a blue whale and often has a large number of remoras attached to his body underneath the dorsal fin.",
+      "Hook is often one of the first whales to arrive at the Sea of Cortez feeding grounds in the Great Whale Conservancies working area in the general vicinity of Loreto, Baja California. He is one of the finest examples of 'habitat preference' coming back to the same part of the sea for many years in a row and a significant number over the past 38 years.",
+      "Artwork by Bruce the Goose",
```

Comment:
I think the artist said that he prefered BruceTheGoose, I think I'll quickly change that. 
```suggestion
      "Artwork by BruceTheGoose",
```